app/src/main/java/org/fossasia/openevent/app/core/faq/list/FaqListPresenter.java

```diff
@@ -18,14 +18,18 @@
 import io.reactivex.schedulers.Schedulers;
 
 import static org.fossasia.openevent.app.common.rx.ViewTransformers.dispose;
+import static org.fossasia.openevent.app.common.rx.ViewTransformers.disposeCompletable;
 import static org.fossasia.openevent.app.common.rx.ViewTransformers.emptiable;
+import static org.fossasia.openevent.app.common.rx.ViewTransformers.progressiveErroneousCompletable;
 import static org.fossasia.openevent.app.common.rx.ViewTransformers.progressiveErroneousRefresh;
 
 public class FaqListPresenter extends BaseDetailPresenter<Long, IFaqListView> {
 
     private final List<Faq> faqs = new ArrayList<>();
     private final IFaqRepository faqRepository;
     private final IDatabaseChangeListener<Faq> faqChangeListener;
+    private Faq previousFaq = new Faq();
+    private Faq currentFaq = new Faq();
```

Comment:
Why allocate new objects?

Reply:
Why do we need a previous faq object. Just compare the incoming one with the current stored one